apps/server/src/modules/board/repo/board-do.repo.spec.ts

```diff
@@ -26,6 +25,7 @@ import {
 	richTextElementFactory,
 	richTextElementNodeFactory,
 } from '@shared/testing';
+import { DrawingElementAdapterService } from '@modules/tldraw-client/service';
```

Comment:
Can you please use a specific barrel files for the module that exact define the export export { DrawingElementAdapterService } from './xxxx'; and use it on all places. I expect you should get a warning for this import.